Loop Intent Extension/OverrideIntentHandler.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  OverrideIntentHandler.swift
+//  Loop Intent Extension
+//
+//  Created by Anna Quinlan on 10/17/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import Intents
+
+class OverrideIntentHandler: NSObject, EnableOverridePresetIntentHandling {
+    lazy var defaults = UserDefaults(suiteName: Bundle.main.appGroupSuiteName)
+    
+    func containsOverrideName(name: String) -> Bool {
+        let lowercasedName = name.lowercased()
+        
+        if let defaults = self.defaults, defaults.intentExtensionInfo?.overridePresetNames?.first(where: {$0.lowercased() == lowercasedName}) != nil {
+            return true
+        }
+        return false
+    }
+    
+    func handle(intent: EnableOverridePresetIntent, completion: @escaping (EnableOverridePresetIntentResponse) -> Void) {
+        guard let defaults = self.defaults, let overrideName = intent.overrideName?.lowercased(), containsOverrideName(name: overrideName) else {
```

Comment:
My (pretty simple) implementation to set overrides just looks for the preset's name (and lowercases the names when comparing to avoid capitalization-related difficulties). This will fail to distinguish between overrides if they have the same name but different capitalization.